oojs.js

```diff
@@ -4,9 +4,28 @@
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
-// Create an instance of the Playlist class.
+class Playlist {
+  constructor(title){
+    this.title = title;
+    this.songs = [];
+    this.favorites = 0;
+  }
+  addSong(song){
+    this.songs.push(song);
 
-// Create an Album class that inherits from Playlist. It should also...
-  // Have an artist property that is determined by some input.
+  }
+}
+// // Create an instance of the Playlist class.
+const sing = new Playlist("loveSong")
+//
+// // Create an Album class that inherits from Playlist. It should also...
+//   // Have an artist property that is determined by some input.
+//
+class Album extends Playlist{
+  constructor(title,artist){
+    this.artists = artist;
```

Comment:
missing the `.super` call in the `Album`'s `constructor` so as to pass the `title` up to the `Playlist`'s constructor

i.e. `super(title)`